### ISSUE TITLE
Some datasets about other tasks....

### ISSUE THREAD

**Issue by lartpang** [lartpang/Segmentation-Saliency-Dataset#15]
## 注意

这里记录一些偶然间碰到的与显著性或者不好归类的数据集.

**Comment by lartpang** [comment]
## MVS-Synth Dataset

### 简单介绍

MVS-Synth Dataset is a photo-realistic synthetic dataset prepared for learning-based Multi-View Stereo algorithms. It consists of 120 sequences, each with 100 frames of urban scenes captured in the video game Grand Theft Auto V.[note] The RGB image, the ground truth depth map, and the camera parameters of each frame are provided.

Compared to other synthetic datasets, MVS-Synth Dataset is more realistic in terms of context and shading, and compared to real-world datasets, MVS-Synth provides complete ground truth disparities which cover regions such as the sky, reflective surfaces, and thin structures, whose ground truths are usually missing in real-world datasets.

### 相关链接

* https://phuang17.github.io/DeepMVS/mvs-synth.html

**Comment by lartpang** [comment]
## GTA-3D Dataset

### 简单介绍

A dataset of 2D imagery, 3D point cloud data, and 3D vehicle bounding box labels all generated using the Grand Theft Auto 5 game engine. The dataset contains image and depth map data captured at 1680x1050 resolution and oriented 3D bounding box labels of all vehicles. It is 55GB in total.

### 相关链接

* https://github.com/oscarmcnulty/gta-3d-dataset

**Comment by lartpang** [comment]
## DIH: Depth Images with Humans

### 简单介绍

The DIH dataset has been created for human body landmark detection and human pose estimation from depth images. Training and deploying good models for these tasks require large amounts of data with high quality annotations. Unfortunately, obtaining precise manual annotation of depth images with body parts is hampered by the fact that people appear roughly as blobs and the annotation task is very time consuming. Synthesizing images provides an easy way to introduce variability in body pose, view perspective and high quality annotations can be easily generated.

However, synthetic depth images does not match real depth data in several aspects: visual characteristics that arise from the depth image generation, i.e. measurement noise, the most problematically depth discontinuity and missing measurements. Hence, real data with annotations is required to fill the detection performance gap this data mismatch can provoke in real data. With this is mind, the DIH dataset also provides real data with annotations for Kinect 2. This real data can be used for both finetuning and testing.

The DIH dataset contains a set of synthetic images and a set of images acquired with a Kinect 2 depth sensor as detailed below. Both dataset contain annotations of 17 body landmarks: head, neck, shoulders, elbows, wrists, hips, knees, ankles and eyes (Figure 1(a)). You are encouraged to see our example code of how to load and visualize the data.

### 相关链接

* https://www.idiap.ch/dataset/dih

**Comment by lartpang** [comment]
## LARGE-SCALE RGB+D DATABASE

### 简单介绍

We introduce an RGB-D scene dataset consisting of more than 200 indoor / outdoor scenes.
This dataset contains synchronized RGB-D frames from both Kinect v2 and Zed stereo camera. For the outdoor scene, we first generate disparity maps using an accurate stereo matching method and convert them using calibration parameters. A per-pixel confidence map of disparity is also provided. Our scenes are captured at various places, e.g., offices, rooms, dormitory, exhibition center, street, road etc., from Yonsei University and Ewha University

This dataset has been used to train convolutional neural networks in our project [1] and for our papers[2], [3], [4], [5]
"High quality 2D-to-multiview contents generation from large-scale RGB-D database", under Grant by the Institute for Information and Communications Technology Promotion(IITP) through the Korean Government(MSIP)(R0115-16-1007)

### 相关链接

* https://dimlrgbd.github.io/

**Comment by lartpang** [comment]
## B-T4SA

### 简单介绍

Much progress has been made in the field of sentiment analysis in the past years. Researchers relied on textual data for this task, while only recently they have started investigating approaches to predict sentiments from multimedia content. With the increasing amount of data shared on social media, there is also a rapidly growing interest in approaches that work ``in the wild'', i.e. that are able to deal with uncontrolled conditions. In this work, we faced the challenge of training a visual sentiment classifier starting from a large set of user-generated and unlabeled contents. In particular, we collected more than 3 million tweets containing both text and images, and we leveraged on the sentiment polarity of the textual contents to train a visual sentiment classifier. To the best of our knowledge, this is the first time that a cross-media learning approach is proposed and tested in this context. We assessed the validity of our model by conducting comparative studies and evaluations on a benchmark for visual sentiment analysis. Our empirical study shows that although the text associated to each image is often noisy and weakly correlated with the image content, it can be profitably exploited to train a deep Convolutional Neural Network that effectively predicts the sentiment polarity of previously unseen images. The dataset used in our experiments, named T4SA (Twitter for Sentiment Analysis), is available on this page.

### 相关链接

* http://www.t4sa.it/

**Comment by lartpang** [comment]
## TAO-VOS

### 简单介绍

We annotated 126 validation sequences of the Tracking Any Object (TAO) dataset with segmentation masks for video object segmentation. Additionally, we annotated all 500 training sequences semi-automatically while ensuring a high quality (for details see paper below). Compared to existing VOS datasets, sequences in TAO-VOS are significantly longer, cover more objects per sequence, and cover more different classes.

### 相关链接

- https://www.vision.rwth-aachen.de/page/taovos

### Citation

If you use this benchmark, please cite

```
@inproceedings{Voigtlaender21WACV,
title={Reducing the Annotation Effort for Video Object Segmentation Datasets},
author={Paul Voigtlaender and Lishu Luo and Chun Yuan and Yong Jiang and Bastian Leibe},
booktitle={WACV},
year={2021}
}
```

and also the original TAO paper

```
@inproceedings{Dave20ECCV,
  title={TAO: A Large-Scale Benchmark for Tracking Any Object},
  author={Achal Dave and Tarasha Khurana and Pavel Tokmakov and Cordelia Schmid and Deva Ramanan},
  booktitle={ECCV},
  year={2020}
}
```

**Comment by lartpang** [comment]
## human gesture and olfactory dimension analysis

- [ ] SniffyArt: The Dataset of Smelling Persons: https://arxiv.org/abs/2311.11888
- [ ] ODOR: The ICPR2022 ODeuropa Challenge on Olfactory Object Recognition: https://arxiv.org/abs/2301.09878

**Comment by lartpang** [comment]
## Fish

- [**FishNet**] FishNet: Deep Neural Networks for Low-Cost Fish Stock Estimation, IEEE COINS 2024, [Paper](https://arxiv.org/pdf/2403.10916), Dataset
- [**Fish4Knowledge**] Supporting Ground-Truth annotation of image datasets using clustering, ICPR 2012, [Paper](https://ieeexplore.ieee.org/document/6460437), [Project](https://groups.inf.ed.ac.uk/vision/DATASETS/FISH4KNOWLEDGE/WEBSITE/GROUNDTRUTH/RECOG/)
- [**DeepFish**] A realistic fish-habitat dataset to evaluate algorithms for underwater visual analysis, Scientific Reports 2020, [Paper](https://doi.org/10.1038/s41598-020-71639-x), [Project](https://alzayats.github.io/DeepFish/)
- A Large-Scale Dataset for Fish Segmentation and Classification, IEEE ASYU 2020, [Paper](https://ieeexplore.ieee.org/abstract/document/9259867), [Dataset (Kaggle)](https://www.kaggle.com/datasets/crowww/a-large-scale-fish-dataset)

## Food

- [**MetaFood3D**] MetaFood3D: Large 3D Food Object Dataset with Nutrition Values, [arXiv:2409.01966](https://arxiv.org/abs/2409.01966)
